bin/data-symlinks

```diff
@@ -18,15 +18,22 @@ for file in "$data"/*/===; do
     perl -pe '$_=lc; s/[^\w]+/-/g; s/^-//; s/-$//'
   )
 
+  baseid=$id
+  if [[ $(basename $id) != $id ]]; then
```

Comment:
Quotes are actually needed for variables on the RHS of a `==` or `!=` in `[[ ... ]]`.

The `shellcheck` program is a great way to find out about these things.